app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedActivityTest.kt

```diff
@@ -750,6 +750,116 @@ class MarkChaptersCompletedActivityTest {
     }
   }
 
+  @Test
+  fun testMarkChaptersCompletedActivity_clickMarkCompleted_activityFinishes() {
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.mark_chapters_completed_mark_completed_text_view)).perform(click())
+      assertThat(activityTestRule.activity).isNull()
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_configChange_clickMarkCompleted_activityFinishes() {
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.mark_chapters_completed_mark_completed_text_view)).perform(click())
+      assertThat(activityTestRule.activity).isNull()
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_selectMultipleChapters_clickMarkCompleted_reopenActivity_completedChaptersAreChecked() { // ktlint-disable max-line-length
```

Comment:
Rename to remove `ktlint` comment.

Reply:
@yashraj-01 Please check the note section - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#how-to-fix-the-most-common-issues

Reply:
Done.